docker-compose.yml

```diff
@@ -2,28 +2,23 @@ version: '3'
 
 services:
   spryker-sdk:
-    image: spryker/php-sdk-local:${SDK_VERSION:-latest}
+    image: spryker/php-sdk-local:latest
     network_mode: "host"
     build:
       context: ./
       dockerfile: ./infrastructure/sdk.local.Dockerfile
       args:
-        USER_UID: ${USER_UID}
-        SDK_VERSION: ${SDK_VERSION:-latest}
+          USER_UID: ${USER_UID}
     working_dir: "/project"
     environment:
       HOST_PWD: ${HOST_PWD}
       EXECUTABLE_FILE_PATH: ${EXECUTABLE_FILE_PATH}
-    env_file:
-      - .env.prod
     volumes:
       - $PWD:/project:rw
-      - ./db:/data/db:rw
-      - ./var:/data/var:rw
-      - ./extension:/data/extension:rw
-      - ./VERSION:/data/VERSION:rw
-      - ./config/packages/workflow.yaml:/data/config/packages/workflow.yaml:rw
-      - ~/.ssh:/home/spryker/.ssh:ro
-      - ./.gitmodules:/data/.gitmodules:rw
+      - .:/data:rw
+#      - ./db:/data/db:rw
```

Comment:
Please avoid single-line comments. We don't need them as long as we have git history.